include tiering configuration in ttl docs (#11704)
ydb-platform/ydb: ydb/docs/en/core/concepts/_includes/ttl.md

> # Time to Live (TTL) and Eviction to External Storage

Reviewer: ```suggestion
# Time to Live (TTL) and eviction to external storage
```

## How it works {#how-it-works}

> The table's TTL is a sequence of storage tiers. Each tier contains an expression (TTL expression) and an action. When the expression triggers, that tier is assigned to the row. When a tier is assigned to a row, the specified action is automatically performed: moving the row to an external storage or deleting it. External storage is represented by the [external data source](https://ydb.tech/docs/en/concepts/datamodel/external_data_source) object. Deleting action can only be specified for the last tier.

Reviewer: ```suggestion
The table's TTL is a sequence of storage tiers. Each tier contains an expression (TTL expression) and an action. When the expression triggers, that tier is assigned to the row. When a tier is assigned to a row, the specified action is automatically performed: moving the row to external storage or deleting it. External storage is represented by the [external data source](../../datamodel/external_data_source.md) object. The delete action can only be specified for the last tier.
```

> {% note info %}

Reviewer: There are too many notes in the text. Let's create a separate section, like "Limitations", and list all these.

> Currently, only an Object Storage is available as external storage.

Reviewer: ```suggestion
Currently, only {{ objstorage-name }} is supported as external storage.
```

{% endnote %}

> {{ ydb-short-name }} allows you to specify a column (TTL column) whose values are used in TTL expressions. The expression triggers when the specified number of seconds have passed since the time recorded in the TTL column.

Reviewer: ```suggestion
{{ ydb-short-name }} allows you to specify a column (TTL column) whose values are used in TTL expressions. The expression triggers when the specified number of seconds has passed since the time recorded in the TTL column.
```

{% endlist %}

> ### Enabling eviction to Object Storage for an existing table {#enable-tiering-on-existing-tables}

Reviewer: The "Concepts" documentation section is for high-level theoretical content. Old practical content is in the process of moving out. So, this content belongs to recipes and/or examples in YQL/SDK reference. Here, only give links there.

{% note info %}

> This functionality is only available for column-oriented tables. For row-oriented tables, this functionality is under development.

Reviewer: We need to create a standardized include for such notes similar to https://github.com/ydb-platform/ydb/blob/main/ydb/docs/en/core/_includes/not_allow_for_olap_note.md (but opposite). Otherwise, such places will all look differently and be harder to find later.

{% endnote %}

> In the following example, rows of the table `mytable` will be moved to the bucket described in the external data source `/Root/s3_cold_data` one hour after the time recorded in the column `created_at`, and will be deleted after 24 hours:

Reviewer: ```suggestion
In the following example, rows of the table `mytable` will be moved to the bucket described in the external data source `/Root/s3_cold_data` one hour after the time recorded in the column `created_at` and will be deleted after 24 hours:
```

{% list tabs group=tool %}

- YQL

  ```yql

>   ALTER TABLE `mytable` SET (TTL = Interval("PT1H") TO EXTERNAL DATA SOURCE `/Root/s3_cold_data`, Interval(PT24H) DELETE ON modified_at AS SECONDS);

Reviewer: ```suggestion
  ALTER TABLE `mytable` SET (
      TTL = 
          Interval("PT1H") TO EXTERNAL DATA SOURCE `/Root/s3_cold_data`,
          Interval("PT24H") DELETE ON modified_at AS SECONDS
   );
```

Author: I formatted the query this way:
```
  ALTER TABLE `mytable` SET (
      TTL =
          Interval("PT1H") TO EXTERNAL DATA SOURCE `/Root/s3_cold_data`,
          Interval(PT24H) DELETE
      ON modified_at AS SECONDS);
```

  ```

{% if oss == true %}

> - C++

Reviewer: What about other programming languages?

Author: Configuration of tiering is not supported in other SDKs, currently
